spec/models/portfolio_spec.rb

```diff
@@ -18,6 +18,23 @@
     end
   end
 
+  context "model fails validation on unique name by same tenant" do
+    let(:portfolio_copy)          { create(:portfolio, :without_tenant) }
+
+    before do
+      ActsAsTenant.current_tenant = tenant
+    end
+
+    it "will fail validation if tenants try to create portfolios with the same name" do
+      portfolio.update(:name => "samename")
+      portfolio_copy.update(:name => "samename")
+
+      expect(portfolio).to be_valid
+      expect(portfolio_copy).to_not be_valid
+      expect(portfolio_copy.errors.messages[:name]).to_not be_nil
+    end
```

Comment:
@lindgrenj6 Please add a second test to validate that the same name can be used when the Portfolio is created in a different tenants.

Reply:
@lindgrenj6 - It would be nice to see a test of the raised exception as well.

Reply:
Added another test to verify different tenants can re-use same name as well as an `expect` clause to make sure an exception is being thrown.